perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -112,16 +111,15 @@ class AwsVirtualMachine(virtual_machine.BaseVirtualMachine):
   imported_keyfile_set = set()
   deleted_keyfile_set = set()
 
-  def __init__(self, vm_spec):
+  def __init__(self, vm_spec, network, firewall):
     """Initialize a AWS virtual machine.
 
     Args:
       vm_spec: virtual_machine.BaseVirtualMachineSpec object of the vm.
```

Comment:
oops did this only on BaseVirtualMachine - I'll do it for all of them - will update in a minute